src/commands/faucet.ts

```diff
@@ -74,14 +74,14 @@ const FaucetModule = (cosmosClient: CosmosClient): Command => {
 
       if (!created) {
         const txHash = drop.get('txHash');
-        await interaction.editReply(`:negative_squared_cross_mark: Each user can only claim once, your past txhash: [${txHash}](${Config.faucet.restUrl}/cosmos/tx/v1beta1/txs/${txHash}).`);
+        await interaction.editReply(`:negative_squared_cross_mark: Each user can only claim once, your past txhash: [${txHash}](https://www.mintscan.io/likecoin/txs/${txHash}).`);
         return;
       }
 
       try {
         const txHash = await cosmosClient.distribute(address);
         await drop.update({ txHash: txHash });
-        await interaction.editReply(`:white_check_mark: Transaction submitted, txhash: [${txHash}](${Config.faucet.restUrl}/cosmos/tx/v1beta1/txs/${txHash})`);
+        await interaction.editReply(`:white_check_mark: Transaction submitted, txhash: [${txHash}](https://www.mintscan.io/likecoin/txs/${txHash})`);
```

Comment:
Can we make this a config where we append `txHash` after it? e.g. `${Config.faucet.chainViewerUrl}${txHash}`
Where `Config.faucet.chainViewerUrl` can be  `https://www.mintscan.io/likecoin/txs/` or `https://node.testnet.like.co//cosmos/tx/v1beta1/txs/`

Reply:
Yeah, was lazy, got caught 🌚